graphql-java-support/src/test/java/com/apollographql/federation/graphqljava/SchemaTransformerTest.java

```diff
@@ -92,4 +94,105 @@ public void getFederatedEntities_noEntities_notFound() {
 
     Assertions.assertTrue(entities.isEmpty());
   }
+
+  @Test
+  public void setSchemaPrinterComparatorRegistry_withDefaultRegistry_succeeds() {
```

Comment:
I'm unsure whats the point of those all new tests as they don't verify anything - lets remove those as it is just noise